tests/profiler/tensorflow2/test_native_tf2_profiler.py

```diff
@@ -0,0 +1,480 @@
+# Standard Library
+import atexit
+import json
+import os
+import pstats
+import time
+from datetime import datetime
+from pathlib import Path
+
+# Third Party
+import pytest
+import tensorflow as tf
+
+# First Party
+import smdebug.tensorflow as smd
+from smdebug.core.collection import CollectionKeys
+from smdebug.profiler.profiler_config_parser import ProfilerConfigParser
+from smdebug.profiler.profiler_constants import (
+    CONVERT_TO_MICROSECS,
+    CPROFILE_NAME,
+    CPROFILE_STATS_FILENAME,
+    DEFAULT_PREFIX,
+    PYINSTRUMENT_HTML_FILENAME,
+    PYINSTRUMENT_JSON_FILENAME,
+    PYINSTRUMENT_NAME,
+    TENSORBOARDTIMELINE_SUFFIX,
+    TRACE_DIRECTORY_FORMAT,
+)
+from smdebug.profiler.python_profile_utils import StepPhase
+from smdebug.profiler.python_profiler import PythonProfiler
+from smdebug.profiler.tf_profiler_parser import TensorboardProfilerEvents
+from smdebug.tensorflow import KerasHook as Hook
+
+
+@pytest.fixture
+def profiler_config_path(config_folder, monkeypatch):
+    config_path = os.path.join(config_folder, "profiler_config.json")
+    monkeypatch.setenv("SMPROFILER_CONFIG_PATH", config_path)
+    yield config_path
+    if os.path.isfile(config_path):
+        os.remove(config_path)
+
+
+def _convert_to_string(item):
+    return '"{0}"'.format(item) if isinstance(item, str) else item
+
+
+def _convert_key_and_value(key, value):
```

Comment:
These functions come from `tests/profiler/core/test_profiler_config_parser.py`. Can we simply import them from that file or move the functions to a utils file for both of the test files to import from?